ext/xml/tests/bug32001.phpt

```diff
@@ -166,6 +166,10 @@ foreach ($suite as $testcase) {
 ?>
 --EXPECTF--
 libxml2 Version => %s
+
```

Comment:
I think, rather than adding the deprecations to the expected output, the test code itself should be changed to be future-proof and deprecation-free.